airflow/dags/common/sps_package.py

```diff
@@ -82,7 +82,7 @@ def acron(self):
     def publisher_id(self):
         try:
             return self.xmltree.xpath(
-                './/article-id[not(@specific-use="scielo") and @pub-id-type="publisher-id"]/text()'
+                './/article-id[not(@specific-use="scielo-v3") and @pub-id-type="publisher-id"]/text()'
```

Comment:
É aceitável que este método retorne um PID na versão 2?

Reply:
@joffilyfe não gosto desta abordagem (`not(@specific-use="scielo-v3")`) porque retonar valor indevido. Vale a pena avaliar o que é o publisher_id, em que contexto que é usado... suponho que deveria `article-id[@specific-use='scielo-v2']`

Reply:
> É aceitável que este método retorne um PID na versão 2?
@gustavofonseca Não tenho certeza. 
---



> @joffilyfe não gosto desta abordagem (`not(@specific-use="scielo-v3")`) porque retonar valor indevido. Vale a pena avaliar o que é o publisher_id, em que contexto que é usado... suponho que deveria `article-id[@specific-use='scielo-v2']`. Além disso, eu já tinha sugerido que o nome `publisher_id` fosse modificado por não é nada explícito o dado que ele representa.

Verificando o código anterior nós podemos observar que essa negativa era feita com o intuito de não retornar o novo `scielo-id` então eu precisei manter a mesma estratégia.

Sobre a semântica do campo `publisher_id` eu não tenho como opinar porque não conheço o contexto de uso destes identificadores, se você puder abrilhantar a discussão com essas informações seria magnifico.

Em relação ao uso dessa variável pelas nossas dags no `opac-airflow` nós podemos ver que ela não está sendo utilizada:

```shell
grep -ri "publisher_id" ./**/*.py
./common/sps_package.py:    def publisher_id(self):
./operations/sync_kernel_to_website_operations.py:    article.pid = _nestget(data, "article_meta", 0, "article_publisher_id", 1)
```

Reply:
então, ou remover ou mudar para o nome scielo_v2 que em algum momento seria usado para isso, não?

Reply:
Eu não havia me atentado ao fato de que este atributo não era utilizado pelo código. Sabendo disto eu sugiro que o código seja removido. O que acha?

Reply:
@joffilyfe @jamilatta @patymori 
fique preocupada com este trecho

`./operations/sync_kernel_to_website_operations.py:    article.pid = _nestget(data, "article_meta", 0, "article_publisher_id", 1)`

Está pegendo dados do article_meta para mandar para o site???

Reply:
> então, ou remover ou mudar para o nome scielo_v2 que em algum momento seria usado para isso, não?

@robertatakenaka acho que o mais correto é remover esse método uma vez que faz mais sentido implementarmos um método `scielo_pid_v2` ([aos moldes do migração](https://github.com/scieloorg/document-store-migracao/blob/master/documentstore_migracao/export/sps_package.py#L136-L137)).

Reply:
@robertatakenaka @gustavofonseca removido em cf3d7307bfb943ab7e99cedd58712482bc987325